app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,56 @@
+module ManagerRefresh::Inventory::Core
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def provider_module
+      @provider_module ||= ManageIQ::Providers::Inflector.provider_module(self)
+    end
+
+    def has_authentications(options = {})
```

Comment:
also you could do this a normal module and then extend instead of include? It would look nicer :-)